src/lang/lexer.ts

```diff
@@ -15,7 +15,7 @@ export class Lexer {
     constructor(source: string) {
         this.source = source
             .replace(/<\?=\s+(\".+\")\s+\?\>/, "{!! $1 !!}")
-            .replace(/(\<\?(php)?\n)(.+)(\n\s+\?\>)/gm, "@php$3 @endphp")
+            .replace(/(\<\?(php)?[\n]*)([\n|\s].+)([\n|\s]+\?\>)/gm, "@php$3@endphp")
```

Comment:
`\s` matches any whitespace characters and stands for `[\r\n\t\f\v ]` in the regex. It would probably be enough to use only `\s`
(Explanation from https://regex101.com)
```suggestion
            .replace(/(\<\?(php)?[\n]*)(\s.+)(\s+\?\>)/gm, "@php$3@endphp")
```

Reply:
Also I am not sure if the first `[\n]*` should also be a `\s*`